### ISSUE TITLE
Handling different averaging periods of TC windspeeds

### ISSUE THREAD

**Issue by ThomasRoosli** [CLIMADA-project/climada_python#748]
Another important aspect to consider here: some agencies report `max_sustained_wind` as 1-minute averages (US agencies) and other as 10-minutes averages (e.g. BOM) or even different.
I suggest that an additional attribute is included in `TCtracks` objects which state the averaging time of this variable, and that this attribute is also taken into account in these algorithms by first rescaling values to the averaging time required by the respective wind models.

_Originally posted by @bguillod in https://github.com/CLIMADA-project/climada_python/issues/694#issuecomment-1528995983_

**Comment by ThomasRoosli** [comment]
An important Reference for that can be found here: Recommended conversion factors between agency estimates of maximum tropical cyclone wind speed given in Table1.2 in Guidelines for converting between various wind averaging periods in tropical cyclone conditions, WMO/TD-No. 1555 https://library.wmo.int/doc_num.php?explnum_id=290

**Comment by tovogt** [comment]
Yes, that's what I was referring.

Note that @ThomasRoosli is proposing to move the rescaling step to the `TropCyclone` class. The `TCTracks` class would only write the averaging period as an attribute. However, the current rescaling in `TCTracks` is not just taking the rescaling period into account, but the reporting WMO agency.

**Comment by ThomasRoosli** [comment]
I agree that the functionality is there, and I do not suggest to move it to TropCyclone class. I only suggest that we add an attribute to TCTracks that specifies how the windspeeds should be interpreted. After `TCTracks.from_ibtracs_netcdf` with `rescale_windspeeds=True`, this attribute would be set to '1-minute sustained winds'. But over other routes (e.g. over `TCForecast` from petals) the attribute might be set differently.
It might be enough if we clearly state that TCTracks should contain only '1-minute sustained winds'.

**Comment by peanutfun** [comment]
`TropCyclone` accepts the windfields as csr matrix or similar array-like structure. I agree that there is little we can do to prevent users from inserting windfields that do not encode "1-minute sustained winds". I also think that this would be an unnecessary restriction. An issue only arises once users apply impact functions that are calibrated on "1-minute sustained winds" to other data, or vice versa. I think we can make this a bit clearer in the documentation (of both `TropCyclone` and `ImpfTropCyclone`), but we need to clarify that users ultimately are responsible for ensuring consistency themselves.

@ThomasRoosli @tovogt do you agree?

**Comment by tovogt** [comment]
I agree that this is about user-friendliness and about improving docs.

@ThomasRoosli Either you have an attribute specifying the wind averaging period, or you just assume that it's always "1-minute sustained winds", and if it's not, then it's a bug. That would mean that the TC forecast IO routines have a bug, namely they fail to convert wind speeds to "1-minute sustained winds".
